test/mosquito/queue_test.cr

```diff
@@ -58,7 +58,7 @@ describe Queue do
     test_queue.enqueue task, in: offset
     stored_time = redis.zscore test_queue.scheduled_q, task.id
 
-    assert_in_delta stored_time.not_nil!, timestamp, 1
+    assert_in_delta stored_time.not_nil!, timestamp, 2
```

Comment:
@alex-lairan can you provide a bit of background on this change?

Reply:
On my computer it taken 2 units instead of 1.

Can remove if not necessary, but it will be red locally.

Reply:
gotcha. Okay, that's probably fine. Perhaps your computer is just doing more things.